src/core/src/package_managers/YumPackageManager.py

```diff
@@ -56,6 +56,14 @@ def __init__(self, env_layer, execution_config, composite_logger, telemetry_writ
         self.set_package_manager_setting(Constants.PKG_MGR_SETTING_IDENTITY, Constants.YUM)
         self.STR_TOTAL_DOWNLOAD_SIZE = "Total download size: "
 
+        # if an Auto Patching request comes in on a CentOS machine with Security and/or Critical classifications selected, we need to install all patches
+        installation_included_classifications = [] if execution_config.included_classifications_list is None else execution_config.included_classifications_list
+        if execution_config.maintenance_run_id is not None and execution_config.operation.lower() == Constants.INSTALLATION.lower() \
+                and 'CentOS' in str(env_layer.platform.linux_distribution()) \
+                and ('Critical' in installation_included_classifications or 'Security' in installation_included_classifications):
+            self.composite_logger.log_debug("Updating classifications list to install all patches for the Auto Patching request since classification based patching is not available on CentOS machines")
+            execution_config.included_classifications_list = [Constants.PACKAGE_CLASSIFICATIONS.__getitem__(1), Constants.PACKAGE_CLASSIFICATIONS.__getitem__(2), Constants.PACKAGE_CLASSIFICATIONS.__getitem__(4)]
```

Comment:
That object was originally created that way to translate an integer representing all classifications (in a bitmask) into the actual list of classifications. In V2, that object is not necessary as it was before and heavily reflects its origin more than what purpose needs to be served today.